index.js

```diff
@@ -115,6 +115,7 @@ function ParseServer(args) {
   router.merge(require('./installations'));
   router.merge(require('./functions'));
   router.merge(require('./schemas'));
+  router.merge(require('./global_config'));
```

Comment:
Whoops, thanks for the clarification about route names. You can check out the network requests from dashboard.parse.com/apps/_/config to see how deleting and modifying config is done. Because of the way browsers/our JS SDK works, the method is passed in a `_method` key in the body of the request, but that is the real method that the server will respond to that one, not the HTTP method.

Once we have an open source dashboard we'll want to verify compatibility using that before we make this non-experimental, so yes I think we should still have the experimental flag.